functions/AutoRemediateOrchestrator.js

```diff
@@ -27,22 +39,54 @@ module.exports.handler = (event, context, callback) => {
     return
   }
 
-  let FunctionName =
+  if (!CONFIG[`${AutoRemediate}`]['delayedExecution']) {
+    console.log('Delayed execution disabled', AutoRemediate)
+    delayedExecution=false
+  }
+  else{
+    delayedExecution=true;
+    console.log('Delayed execution enabled', AutoRemediate)
+    delay=CONFIG[`${AutoRemediate}`]['delay']
+    queueURL='https://sqs.us-west-2.amazonaws.com/747218156759/CloudConformityAutoRemediate'
```

Comment:
We shouldn't leave the aws account in. We can use environment variable for queue url . 
for example
```
queueURL= process.env.QUEUE_URL
```